gci/models.py

```diff
@@ -1 +1,30 @@
-# Create your models here.
+from django.db import models
+
+class Students(models.Model):
+    student_id = models.IntegerField(default=0, primary_key=True)
+    student_name = models.CharField(max_length=100, default='')
+    program_year = models.ForeignKey('ProgramYear', null=True)
+
+class Organizations(models.Model):
+    org_id = models.IntegerField(default=0, primary_key=True)
+    org_name = models.CharField(max_length=100, default='')
+    org_logo = models.ImageField(upload_to='', blank=True, default='')
+    org_url = models.URLField(default='', blank=True)
+    org_summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Students)
+
+class Tasks(models.Model):
+    task_id = models.IntegerField(default=0, primary_key=True)
+    task_name = models.CharField(max_length=100, default='')
+    languages_req = models.TextField(max_length=200, default='')
+    task_summary = models.TextField(max_length=300, default='')
```

Comment:
this length is wrong
refer to the [spec](https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf)!